hw/top_earlgrey/data/chip_testplan.hjson

```diff
@@ -2983,5 +2983,121 @@
+            - for first pass simplicity can activate IO portion only for now through GPIO
+          - for dedicated pins, focus on SPI device quad activity
+          - USB activity should be activated
+            - for first pass simplicity activate IO portion only for now via pin forcing in usbdev.
+        - Ongoing cpu activity (icache / SRAM scrambling both activated)
+          - servicing ongoing threads and random read/write data to memory
+          - icache needs to be activated, otherwise the system may spend most of its time fetching code
+        - Background checks enabled wherever possible
+          - rstmgr background checks
+          - alert_handler ping checks
+          - OTP background checks
+
+      This test should leverage the OTTF test framework for supporting
+      concurrency in a FreeRTOS environment.
+      '''
+      stage: V2
+      tests: []
+    }
+    {
+      name: chip_sw_power_idle_load
+      desc: '''Concurrency test modeling load conditions in idle state
+
+      This concurrency test models an average idle scenarios.
+
+      The test should be made configurable so that the type of power state and
+      the time spent in a particular power state can be configured via a
+      flag (or similar). This will make it easier to reuse the test for power
+      simulation and characterization later on.
+
+      The test should cover the following scenarios:
+
+        - Processor is in WFI state
+        - Background checks enabled wherever possible
+          - rstmgr background checks
+          - alert_handler ping checks
+          - OTP background checks
+        - Timers (regular and AON) are active
+        - TBD: check whether transactional clocks should be enabled or disabled.
+        - TBD: check whether PWM should be active.
+
+      This test should leverage the OTTF test framework for supporting
+      concurrency in a FreeRTOS environment.
+      '''
+      stage: V2
+      tests: []
+    }
+    {
+      name: chip_sw_power_sleep_load
+      desc: '''Concurrency test modeling load conditions in idle state
+
+      This concurrency test models average sleep scenarios.
+
+      The test should be made configurable so that the type of power state and
+      the time spent in a particular power state can be configured via a
+      flag (or similar). This will make it easier to reuse the test for power
+      simulation and characterization later on.
+
+      The test should cover the following scenarios:
+
+        - System can be in deep or light sleep
+        - The system has the following AON / IO activity:
+          - aon_timer active
+          - adc_ctrl active in low power mode
+          - TBD: check whether sysrst_ctrl and pinmux wakeup detectors should be active
+          - TBD: check whether PWM should be active
+
+      This test should leverage the OTTF test framework for supporting
+      concurrency in a FreeRTOS environment.
+      '''
+      stage: V2
+      tests: []
```

Comment:
for this particular one, do you want to x-ref the tests the tests that are already done?

Reply:
Yeah this should be x'refd with tests that are already completed.

Reply:
sounds good.

Reply:
Which test would be a good one to extend here? `chip_sw_pwrmgr_deep_sleep_all_wake_ups`?